core/src/test/java/org/apache/brooklyn/core/entity/ApplicationLoggingTest.java

```diff
@@ -97,12 +91,13 @@ protected void doStop() {
     public static final class TestEntityWithLoggingImp extends TestEntityImpl implements TestEntityWithLogging {
```

Comment:
I think you intended this class to be named `TestEntityWithLoggingImpl`. The 'L' at the end is misisng.